src/python/dxpy/scripts/dx.py

```diff
@@ -3085,7 +3119,7 @@ def run_body(args, executable, dest_proj, dest_path, preset_inputs=None, input_n
 
     if run_kwargs["priority"] in ["low", "normal"] and not args.brief:
         special_access = set()
-        executable_desc = executable_describe or executable.describe()
+        executable_desc = executable.describe()
```

Comment:
Before we used the `executable_describe` as a cache that we could re-use in case we already called `describe` on the executable. Now we make the request twice (potentially, i.e. when the if statements are true). Is there a reason we don't need this caching it any more?

Reply:
added. Thanks for the suggestion :D